charts/localstack/templates/_helpers.tpl

```diff
@@ -23,6 +23,19 @@ If release name contains chart name it will be used as a full name.
 {{- end }}
 {{- end }}
 
+{{/*
+Create a default namespace for the app.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If a common.names.namespace is provided, it will be used as the namespace.
+*/}}
+{{- define "common.names.namespace" -}}
+{{- if .Values.common.names.namespaces }}
+{{- .Values.common.names.namespaces | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- .Release.Namespace | quote }}
+{{- end }}
+{{- end }}
+
```

Comment:
I think there's a bit of a misunderstanding. My point in the previous comments is that we can just directly use `common.names.namespace` (exactly the same way as the Novu chart) because this is defined in the bitnami commons (which we also have as a dependency) here: https://github.com/bitnami/charts/blob/8ecfbadb2b93576772fd134aeedcb13ad20221d2/bitnami/common/templates/_names.tpl#L59-L64

So you can just remove this section, just use the (already by the subchart defined) `common.names.namespaces` everywhere (as you are already 🥳), and use the value `namespaceOverride` in the `values`.

```suggestion
```

Reply:
Alright, I have just pushed the changes as you suggested. However, I am unclear about the instruction to 'use the value namespaceOverride in the values'. Could you please provide some clarification?

Reply:
What do you mean with "Novu will not permit us to update it"? Can't you just set arbitrary values for transitive charts? Wouldn't it just be `novu.localstack.namespaceOverride`?